internal/output/table.go

```diff
@@ -51,14 +51,31 @@ func ToTable(out Root, opts Options) ([]byte, error) {
 
 		s += "\n"
 
-		tableOut := tableForBreakdown(out.Currency, *project.Breakdown, opts.Fields, includeProjectTotals)
+		if len(project.Metadata.Errors) > 0 {
```

Comment:
Should we have the same logic in the diff output? The other option would be something like:
```
The following projects could not be evaluated:
infracost/infracost/cmd/infracost/testdata/breakdown_multi_project_with_error/dev (Module path: dev)
Run the following command to see more details: infracost breakdown --path=/path/to/code
```

Reply:
sounds good